importer/v8/importer.go

```diff
@@ -144,42 +144,49 @@ func (i *Importer) Import() error {
 	return nil
 }
 
-func (i *Importer) processDDL(scanner *bufio.Scanner) {
-	for scanner.Scan() {
-		line := scanner.Text()
+func (i *Importer) processDDL(scanner *bufio.Reader) {
+	for {
+		line, _, err := scanner.ReadLine()
+		if err == io.EOF {
+			break
+		}
 		// If we find the DML token, we are done with DDL
-		if strings.HasPrefix(line, "# DML") {
+		if strings.HasPrefix(string(line), "# DML") {
 			return
 		}
-		if strings.HasPrefix(line, "#") {
+		if strings.HasPrefix(string(line), "#") {
 			continue
 		}
 		// Skip blank lines
-		if strings.TrimSpace(line) == "" {
+		if strings.TrimSpace(string(line)) == "" {
 			continue
 		}
-		i.queryExecutor(line)
+		i.queryExecutor(string(line))
 	}
 }
 
-func (i *Importer) processDML(scanner *bufio.Scanner) {
+func (i *Importer) processDML(scanner *bufio.Reader) {
 	start := time.Now()
-	for scanner.Scan() {
-		line := scanner.Text()
-		if strings.HasPrefix(line, "# CONTEXT-DATABASE:") {
-			i.database = strings.TrimSpace(strings.Split(line, ":")[1])
+	for {
+		line, _, err := scanner.ReadLine()
```

Comment:
You should take a similar approach to the one I suggested previously in the review.